README.md

```diff
@@ -2,69 +2,70 @@
 
 [![Build Status SVG][]][Repo on GitHub Actions]
 [![OpenSSF Scorecard SVG][]][Scorecard Results]
-[![first-timers SVG][]][first-timers]
 
-The https://dart.dev site, built with [Jekyll][] and hosted on [Firebase][].
+The documentation site for the [Dart programming language](https://dart.dev), 
+built with [Eleventy][] and hosted on [Firebase][].
 
-[We welcome contributions](CONTRIBUTING.md),
-and we're [first-timer friendly][first-timers]!
```

Comment:
We are! But when I was updating the README I realized the label isn't there anymore, so I removed this link for now to avoid any confusion.

I have follow-up work to introduce more contributing docs for the site infra, syntax, and writing practices. Once that's done and this setup is finalized, I'll bring something back here so it's actually easy to get going as a "first-timer".